frontend/src/app/shared/directives/app-save-on-typing.directive.ts

```diff
@@ -42,7 +43,7 @@ export class AppSaveOnTypingDirective implements OnDestroy {
                 }),
                 // Keeping a key (like backspace) pressed for a more than a certain threshold will result in many key presses
                 // The debounceTime should be above that threshold to not register the initial keypress as the first update
-                debounceTime(600),
+                mergeMap(value => isBeingTestedByCypress() ? of(value) : of(value).pipe(debounceTime(600))),
```

Comment:
Is this the correct way of conditionally piping filters? I'm far from an expert of observables.

Reply:
Because `isBeingTestedByCypress()` should not change during the execution of the of the observable, this should suffice:

```ts
pipe(
...,
isBeingTestedByCypress() ? debounceTime(600) : tap(),
...
)
```

Though, I would reconsider whether this is a good idea, as it not just logs stuff to cypress but changes the behavior, making the e2e tests less useful.

Reply:
Alternatively, I could add a `cy.wait(600)` to every assertion if an event from an input field was fired. I'd figured that faster tests are more important than the debounce time.

Reply:
But thank you, that was exactly what I was looking for, I just could not find the _identity_ filter